heroku/cli
# fix: apps:rename and apps:destroy incorrectly handles git remotes

fixes #3109 

## To test
1. Add 1 or more remotes that point to the same app. e.g. `heroku git:remote APP_NAME && heroku git:remote APP_NAME` --remote heroku-dev` - This will add 2 remotes
2. Verify `heroku` and `heroku-dev`  exist and they both point to the same URL
3. Run `./bin/run apps:rename -a OLD_APP_NAME NEW_APP_NAME` with the cwd of your local git repo.
4. Verify both remotes have changed `git remote -v`
